pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.tsx

```diff
@@ -127,77 +128,72 @@ export const Table: FC<TableProps> = ({
     onSortChange((state as any).sortBy);
   }, [(state as any).sortBy]);
 
+  const RenderHeader = (headerGroup: HeaderGroup) => (
+    <div {...headerGroup.getHeaderGroupProps()} className={cx('tr', styles.headerRow)}>
+      {headerGroup.headers.map((column: any) => {
+        const { HeaderAccessor } = column;
+
+        column.Header = () => HeaderAccessor();
+        let sorted = '';
+
+        if (column.isSorted) {
+          if (column.isSortedDesc) {
+            sorted = 'desc';
+          } else {
+            sorted = 'asc';
+          }
+        }
+
+        return (
+          <div {...column.getHeaderProps()} className="th">
```

Comment:
I guess that the hardcoded class here is because of antd, would it be possible to move that style to an emotion css style? Does it require too much effort?

If we start doing this we can gradually get rid of antd.

Reply:
There is no antd in this table component )

Reply:
Yes it would require some effort look at Table.styles file and everything is based on this.